### ISSUE TITLE
Request : Interactive [scroll-able] Tibian Map

### ISSUE THREAD

**Issue by Sinsae** [Galarzaa90/NabBot#99]
This request is on something I mentioned previously on the official help Discord; and interactive map, that can scroll through the Tibian map, by the use of 'emoji reactions'. 

**↖ ↑   ↗
←   ∙    →
↙   ↓   ↘**

I think enough's been said.
And you know all about how to make it happen too ><
It would take me month's on this simple thing, compared to a decent night's <probably way less> work on your end ;S
Still a nice idea.

**Comment by Galarzaa90** [comment]
One of the drawbacks with reactions is their response time, due to API rate-limiting. You must have noticed already how when you use a command that has vocation filters, a lot of the time the reactions take a bit to appear. That's because of the API's rate limit, making the bot wait for the cooldown to expire.

So, I would have to drop the diagonal emojis, because reactions are needed for up and down too.

⬅️ ⬆️ ➡️ ⬇️ 🔼 🔽  (those red ones are all blue in discord)

So, navigation would be kind of clunky, because it would advance in sectors, so if you want to point an area that ended up right on a corner of the map, you won't be able to do much about it.

It is doable, but it has many complications. It could probably be used as a way to show coordinates in discord, like parse a link from tibiawiki and show it in discord.

**Comment by Sinsae** [comment]
Exactly, >< , that's why I know my limitations vs. yours, hahah. I won't be a tibia wiki map <draggable> by any means, but for a Tibian Discord server, it will be the first of it's kind. And Mr Galarzaa, will be the one who makes it. >xD
There could also be maybe 4 more emoji's where you can 'snap-to' the top right/left or bottom right/left.
Depending on the discord emoji's to make it less complicated. But it will still be the first of it's kind.

**Comment by Sinsae** [comment]
Until I make classes, I can only make suggestions ;*
